src/views/HomeView/HomeView.js

```diff
@@ -1,56 +1,29 @@
 import React, { useState, useEffect } from 'react';
-import PropTypes from 'prop-types';
 
 import './HomeView.css';
 
 import UserInteractions from '../../components/UserInteractions/UserInteractions';
-import FormModal from '../../components/FormModal/FormModal';
 import Feed from '../../components/Feed/Feed';
+import { user } from '../../Data/mockData';
 
 const HomeView = () => {
+  const [currentUser, setCurrentUser] = useState({})
   const [isHome, setIsHome] = useState(false)
-  // const [allPosts, setAllPosts] = useState([]);
-  // const [userLocation, setUserLocation] = useState({});
-  // const [error, setError] = useState('');
-
-  // const getUserLocation = async () => {
-  //   // update with URL for geocoding location API
-  //   const url = '';
-  //   setError('');
-  //
-  //   try {
-  //     const response = await fetch(url)
-  //     const userLocation = await response.json()
-  //     setUserLocation(userLocation);
-  //   } catch(error) {
-  //     setError(error.message);
-  //   }
-  // }
-  //
-  // const getAllPosts = async () => {
-  //   // update with URL to BE to request from when avail.
-  //   const url = '';
-  //   setError('');
-  //
-  //   try {
-  //     const response = await fetch(url)
-  //     const allPosts = await response.json()
-  //     setAllPosts(allPosts);
-  //   } catch(error) {
-  //     setError(error.message);
-  //   }
-  // }
 
   useEffect(() => {
+    setCurrentUser(user)
     setIsHome(true)
-    // getUserLocation();
-    // getAllPosts();
     }, [])
 
   return (
     <section className='home-view'>
-      <UserInteractions isHome={isHome}/>
-      <Feed />
+
+      <section className='feed-container'>
+        <UserInteractions isHome={isHome} user={currentUser}/>
+        <h3 className='feed-title'>All Posts</h3>
```

Comment:
Nice, this will look more consistent with the profile view now.

Reply:
Yeah exactly - I'd left the HomeView a bit messy when I waded into the CI config process, so I appreciated being able to follow some of what you'd set up in ProfileView and translating it to work for the Home page as well.